### ISSUE TITLE
Events Diary - "Select Date" dialogue buttons are in unclear positions, wording unclear

### ISSUE THREAD

**Issue by ACascarino** [backstagetechnicalservices/website#59]
The "Select Date" dialogue features two buttons, one reading "Select date" and the other "Go to this month", as pictured below:

![image](https://user-images.githubusercontent.com/14007028/42480895-aecee00c-83d7-11e8-9cf5-a72680e5052b.png)

Given that whenever a popup dialogue is used throughout the rest of the website the "confirm" or "continue" option is always in the very bottom right corner, as demonstrated below, it seems counterintuitive for that not to be the case here:

![image](https://user-images.githubusercontent.com/14007028/42481137-c995184c-83d8-11e8-98a5-a6305193cae4.png)

![image](https://user-images.githubusercontent.com/14007028/42481300-860707a6-83d9-11e8-87b1-529b95b8e18d.png)

![image](https://user-images.githubusercontent.com/14007028/42481317-ad892bba-83d9-11e8-981c-e9110f124c60.png)

I'd propose swapping them, as below:

![image](https://user-images.githubusercontent.com/14007028/42481489-90345908-83da-11e8-98e2-4a4612f2038a.png)

Additionally, the wording "Go to this month" seems a little unclear - I've in the past interpreted it as "go to the month you've chosen", which further compounds the "go forward action isn't in the bottom right" issue. Something more akin to "Go to present month" might be clearer, or simply "Go to present".

(As an aside, was it formerly the case that you could access this dialogue by clicking on the month displayed? I quite liked that :( )

**Comment by jordanlkirk99** [comment]
I agree with swapping the buttons around, but for the wording I'd prefer "Go to Today" - shorter and simpler.

However I'd prefer to use the date picker used elsewhere in the website, but set to pick months. This would be more consistent and a look better than a couple drop of down boxes.
![image](https://user-images.githubusercontent.com/40248045/42504772-aae58a4c-8434-11e8-9d64-ece4affeb435.png)

**Comment by bnjns** [comment]
I don't think the button positioning really matters, but agreed on the wording. I like Jordan's idea - hadn't even crossed my mind to do that! I think the plan will be to change it to use the datepicker in month/year mode and move it to a drop-down rather than a modal.
